### PR TITLE
added avatar component

### PR DESCRIPTION
**Pull request by varnita619**

### CODE REVIEW

**Review by mansipadgelwar — APPROVED** [review]

File: Components/avatar/avatar.html [hunk 1]
```diff
@@ -129,28 +130,59 @@ <h4 class="list-topic"> Components</h4>
...
+                                alt="" class="round-avatar-image avatar-lg">
+                        </div>
+                    
+                        <div class="avatar">
+                            <img src="https://images.unsplash.com/photo-1606663889134-b1dedb5ed8b7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+                                alt="" class="round-avatar-image avatar-md">
+                        </div>
+                    
+                        <div class="avatar">
+                            <img src="https://images.unsplash.com/photo-1635586409095-b5d87cebe12b?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
```
Comment (anuj-kumary) [inline Components/avatar/avatar.html]: Always give **alt** (alternate) in (avatar, badge).

**Review by pritam-kr — APPROVED** [review]

File: Components/avatar/avatar.html (hunk 1 above)
Comment (varnita619) [reply Components/avatar/avatar.html]: Added that

File: Components/alert/alert.css [hunk 2]
```diff
@@ -17,47 +17,47 @@
...
+    background-color: var(--secondary-color);
     border-radius: 7px;
 }
 
-.success-alert{
+.success-alert {
     width: 100%;
     padding: 1rem 3rem 1rem 1rem;
     margin: 1rem 0;
     font-size: 1rem;
```
Comment (Maianki) [inline Components/alert/alert.css:42]: All alerts have the same width, padding, margin, font size, and border radius. You can create a single class called alert which will contain all these common properties and use that class along with other specific classes for each alert.
For e.g 
``` 
<div class="alert success-alert"> </div>
```

File: Components/avatar/avatar.css [hunk 3]
```diff
@@ -1,41 +1,49 @@
...
+.avatar-xl {
+    width: 8rem;
     height: 8rem;
 }
 
-.avatar-lg img{
-    width: 5rem;
+.avatar-lg {
+    width: 6rem;
     height: 6rem;
```
Comment (Maianki) [inline Components/avatar/avatar.css:38]: Try being consistent with indentation. It will help in readability.

**Review by Maianki — CHANGES_REQUESTED** [review]

**Pull request merged**